plugins/GymFactory/src/GymFactory.cpp

```diff
@@ -74,8 +74,9 @@ gympp::EnvironmentPtr gympp::GymFactory::make(const std::__cxx11::string& envNam
     // Create the environment
     auto ignGym = std::make_shared<gazebo::IgnitionEnvironment>(actionSpace,
                                                                 observationSpace,
-                                                                /*updateRate=*/50,
-                                                                /*iterations=*/10);
+                                                                /*updateRate=*/10000000,
```

Comment:
This number if just an arbitrary big number?

Reply:
Yes, it is to make the simulation go as fast as possible. I saw a similar trick in ign-gazebo.